tools/sylph/macros.xml

```diff
@@ -0,0 +1,55 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.6.1</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@LICENSE@">MIT</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">sylph</requirement>
+            <requirement type="package" version="3.11.9">python</requirement>
+            <requirement type="package" version="2.2.3">pandas</requirement>
+        </requirements>
+    </xml>
+    <xml name="description">
+        <description>fast and precise species-level metagenomic profiling with ANIs</description>
+    </xml>
+    <xml name="citation">
+        <citations>
+            <citation type="doi">10.1038/s41587-024-02412-y</citation>
+        </citations>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <organization name="University of Toronto" url="https://github.com/bluenote-1577/sylph"/>
+        </creator>
+    </xml>
+    <xml name="xrefs">
+        <xrefs>
+            <xref type="bio.tools">sylph</xref>
+        </xrefs>
+    </xml>
+    <xml name="input_database">
+        <conditional name="database_select">
+            <param name="select" type="select" label="Choose the source for databases and metadata">
+                <option value="cached">Cached data</option>
+                <option value="history">History</option>
+            </param>
+            <when value="cached">
+                <param label="Select a sylph database" name="sylph_database" type="select">
+                    <options from_data_table="sylph_databases">
+                        <validator message="No Sylph databases are available" type="no_options" />
```

Comment:
add here a filter for version `1` or something like that ... and then we include version 1 in the test file.

Whenever the tool changes to DB layout we increase this version.

Reply:
ping, you are not filtering the DB here according to the version